packages/realm/src/scripts/submit-analytics.ts

```diff
@@ -309,13 +304,11 @@ async function collectPlatformData(packagePath = getProjectRoot()) {
  * Collect and send analytics data to MongoDB over HTTPS
  * If `REALM_DISABLE_ANALYTICS` is set, no data is submitted to MongoDB
  */
-async function submitAnalytics() {
+export async function submitAnalytics() {
   const data = await collectPlatformData();
   const payload = {
-    webHook: {
-      event: "install",
-      properties: data,
-    },
+    event: "install",
+    properties: data,
```

Comment:
No need to wrap this in a `webHook` object, since it was always used as `payload.webHook`.